### DISCUSSION TITLE
Crusader via starlink

### DISCUSSION THREAD

**Discussion by dtaht** [Zoxc/crusader#82] [Wi-Fi routers and access points]
crusader test --load-duration 60 --streams 8 --stream-stagger 4 flent-fremont.bufferbloat.net (some of these were to Dallas, also)

![plot 2024 09 30 14-17-02](https://github.com/user-attachments/assets/7e49e18e-3982-449a-985a-1e9de2da6fc2)

![plot 2024 09 30 14-07-47](https://github.com/user-attachments/assets/5ed1a7ef-f9c8-46ea-9985-8a9120c43042)

![plot 2024 09 30 14-01-49](https://github.com/user-attachments/assets/8249bf48-57d5-4d77-983a-c0210db61023)

![plot 2024 09 30 13-55-00](https://github.com/user-attachments/assets/2118f628-7598-4501-84cd-43fbcfce8b6d)

**Comment by Zoxc** [comment]
There's no Wi-Fi involved here right? Those latency spikes do remind me of Wi-Fi. You could probably bump your client version to 0.3.1 to get some stats too.

**Comment by dtaht** [comment]
there is wifi to the router but it rarely matters with bandwidths this low. starlink changes characteristics roughly once every 15 seconds, so your typical 24 second test is going to have one bit of noise in it, which is why I go out the 60 (240) seconds.

And 0.3.1 has only been out for 38 minutes... :) I tried out the new transferred plot just now:

![test 2024-09-30 15 52 14](https://github.com/user-attachments/assets/e37462cc-3262-4ced-9b0a-4cb8d7e9bad2)

It's dumping output into crusader-results now.

**Reply by dtaht** [reply]
./crusader test --load-duration 60 --streams 8 --stream-stagger 4 --plot-transferred flent-fremont.bufferbloat.net

**Comment by dtaht** [comment]
![test 2024-09-30 16 01 52](https://github.com/user-attachments/assets/f4292310-9b30-43ea-ac8a-5581ca64f219)
